build.rs

```diff
@@ -230,13 +230,29 @@ fn parse_header_defines(header_path: impl AsRef<Path>) -> Result<HashMap<String,
     Ok(definitions)
 }
 
-fn fix_config_map(mut config_map: HashMap<String, String>) -> HashMap<String, String> {
+fn fix_config_map(
+    mut config_map: HashMap<String, String>,
+    interpreter_config: &InterpreterConfig,
+) -> HashMap<String, String> {
+    if interpreter_config.version.major == 3 && interpreter_config.version.minor.unwrap_or(0) > 7 {
+        if let Some("1") = config_map.get("Py_DEBUG").as_ref().map(|s| s.as_str()) {
+            config_map.insert("Py_REF_DEBUG".to_owned(), "1".to_owned());
+        }
+        if let Some("1") = config_map.get("Py_TRACE_REFS").as_ref().map(|s| s.as_str()) {
+            config_map.insert("Py_TRACE_REFS".to_owned(), "1".to_owned());
+        }
+        // find CFLAGS to whether contains COUNT_ALLOCS then set it as 1
+        // if interpreter_config.version.minor.unwrap_or(0) < 9 {
+        // }
```

Comment:
Dead code?

```suggestion
```

Reply:
Since COUNT_ALLOCS used rarely, leave it unimpl now.

Reply:
Let's delete this code then. `COUNT_ALLOCS` will still work anyway; this `fix_config_map` function was only trying to add `COUNT_ALLOCS` if `Py_Debug` was set.

Reply:
This code is for 3.8 only. according the CFlags to decide whether to set COUNT_ALLOCS as 1., no by the Py_DEBUG, 
Again, Of course, if  we decide never support COUNT_ALLOCS( we know >= 3.9 have remove COUNT_ALLOCS  this code), we just delete this comment.